### DISCUSSION TITLE
[Bug]: Generative AI descriptions are using literal "{label}" text instead of variable replacement

### DISCUSSION THREAD

**Discussion by nwish** [blakeblackshear/frigate#14251] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

Generative AI descriptions are inserting the literal word "label" surrounded by curly braces "{label}" instead of replacing with actual label variable.

![image](https://github.com/user-attachments/assets/8d7d0afc-23b8-4c13-951b-d782b8f84e0b)

### Steps to reproduce

1. Go to UI
2. Go to Explore page
3. Click on a tracked object
4. View GAI description

### Version

0.15.0-a2f42d5

### In which browser(s) are you experiencing the issue with?

Latest Chrome

### Frigate config file

```yaml
.
```

### docker-compose file or Docker CLI command

```yaml
.
```

### Relevant Frigate log output

```shell
.
```

### Relevant go2rtc log output

```shell
.
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Various

### Screenshots of the Frigate UI's System metrics pages

.

### Any other information that may be helpful

.

**Comment by hawkeye217** [comment]
What is your frigate config?

**Comment by hawkeye217** [comment, marked as answer]
This should be fixed by https://github.com/blakeblackshear/frigate/pull/14256
